native/cocos/bindings/jswrapper/v8/MissingSymbols.cpp

```diff
@@ -22,6 +22,8 @@
  THE SOFTWARE.
 ****************************************************************************/
 
+#include "../config.h"
+
 #if SCRIPT_ENGINE_TYPE == SCRIPT_ENGINE_V8
```

Comment:
SCRIPT_ENGINE_V8 is defined in `config.h`, if it isn't included, it will be an undefined value.